docs/tutorials/basic/data.md

```diff
@@ -192,17 +128,390 @@ class SimpleIter:
+s1 = mx.recordio.pack(header1, data)
+print('float label:', repr(s1))
+label2 = [1.0, 2.0, 3.0]
+header2 = mx.recordio.IRHeader(flag=0, label=label2, id=2, id2=0)
+s2 = mx.recordio.pack(header2, data)
+print('array label:', repr(s2))
+```
 
-- `image_iter` by `ImageRecordIter` so we can enjoy the provided multithreaded
-pre-fetch and augmentation.
-- `caption_iter` by `NDArrayIter` or bucketing iterator provided in the `rnn`
-package.
+    ('float label:', "'\\x00\\x00\\x00\\x00\\x00\\x00\\x80?\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00data'")
+    ('array label:', "'\\x03\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x02\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x80?\\x00\\x00\\x00@\\x00\\x00@@data'")
 
-Next we create an iterator whose `next()` function will both
-`image_iter.next()` and `caption_iter.next()` and return the combined results.
 
 
+```python
+# `unpack`
+print(mx.recordio.unpack(s1))
+print(mx.recordio.unpack(s2))
+```
+
+    (HEADER(flag=0, label=1.0, id=1, id2=0), 'data')
+    (HEADER(flag=3, label=array([ 1.,  2.,  3.], dtype=float32), id=2, id2=0), 'data')
+
+
+
+#### Packing/Unpacking Image Data.
+
+*MXNet* provides [__`pack_img`__](http://mxnet.io/api/python/io.html#mxnet.recordio.pack_img) and [__`unpack_img`__](http://mxnet.io/api/python/io.html#mxnet.recordio.unpack_img) to pack/unpack image data. 
+Records packed by `pack_img` can be loaded by `mx.io.ImageRecordIter`.
+
+
+```python
+data = np.ones((3,3,1), dtype=np.uint8)
+label = 1.0
+header = mx.recordio.IRHeader(flag=0, label=label, id=0, id2=0)
+s = mx.recordio.pack_img(header, data, quality=100, img_fmt='.jpg')
+print(repr(s))
+```
+
+    '\x00\x00\x00\x00\x00\x00\x80?\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xd8\xff\xe0\x00\x10JFIF\x00\x01\x01\x00\x00\x01\x00\x01\x00\x00\xff\xdb\x00C\x00\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\xff\xc0\x00\x0b\x08\x00\x03\x00\x03\x01\x01\x11\x00\xff\xc4\x00\x1f\x00\x00\x01\x05\x01\x01\x01\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x01\x02\x03\x04\x05\x06\x07\x08\t\n\x0b\xff\xc4\x00\xb5\x10\x00\x02\x01\x03\x03\x02\x04\x03\x05\x05\x04\x04\x00\x00\x01}\x01\x02\x03\x00\x04\x11\x05\x12!1A\x06\x13Qa\x07"q\x142\x81\x91\xa1\x08#B\xb1\xc1\x15R\xd1\xf0$3br\x82\t\n\x16\x17\x18\x19\x1a%&\'()*456789:CDEFGHIJSTUVWXYZcdefghijstuvwxyz\x83\x84\x85\x86\x87\x88\x89\x8a\x92\x93\x94\x95\x96\x97\x98\x99\x9a\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xb2\xb3\xb4\xb5\xb6\xb7\xb8\xb9\xba\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xd2\xd3\xd4\xd5\xd6\xd7\xd8\xd9\xda\xe1\xe2\xe3\xe4\xe5\xe6\xe7\xe8\xe9\xea\xf1\xf2\xf3\xf4\xf5\xf6\xf7\xf8\xf9\xfa\xff\xda\x00\x08\x01\x01\x00\x00?\x00\xfe\x01\xeb\xff\xd9'
```

Comment:
As discussed earlier, this doesn't seem to be adding too much value and so we can remove it.